src/sandstorm/run-bundle.c++

```diff
@@ -1183,18 +1189,17 @@ private:
     KJ_SYSCALL(sigprocmask(SIG_SETMASK, &sigset, nullptr));
   }
 
-  void copyEtc() {
+  void linkEtc() {
     auto files = splitLines(readAll("etc.list"));
```

Comment:
Darn, this patchset can't handle that setup.  I'm also not sure how we would, short of handling that case specifically (by, say, readlink -f each of the things in `etc.list`, and making *that* the target instead, but that will probably break in other setups)  `/etc/resolv.conf` points to a folder under `/etc/`, but we need it to point to that folder under `/etc.host`.